siem.py

```diff
@@ -64,6 +64,8 @@
     "location": "dhost",
 }
 
+CONVERT_DHOST_FIELD_TO_VALID_FQDN = True
```

Comment:
Please avoid arbitrary global variables. Why not pass config_data down into the write_* functions and into the update_cef_keys function?

Reply:
Done